### ISSUE TITLE
save changes dialog on app close.

### ISSUE THREAD

**Issue by amilcarlucas** [ArduPilot/MethodicConfigurator#53]
**Describe the bug**
When closing the app parameter changes get silently discarded.

**Expected behavior**
A pop-up asking if the parameter changes should be written to disk and/or FC.

Submited by Yuri.

**Comment by amilcarlucas** [comment]
@yuri-rage sorry I tested this and there is no bug.
If a parameter is changed it already asks the user if he wants to save the changes to file.

Closing this

**Comment by yuri-rage** [comment]
Agree. I misinterpreted that dialog in my abundance of caution over saving files.
